nav2_waypoint_follower/src/waypoint_follower.cpp

```diff
@@ -215,6 +218,8 @@ WaypointFollower::followWaypoints()
     }
 
     feedback->current_waypoint = goal_index;
+    const auto current_time = this->now();
+    feedback->elapsed_time = current_time - start_time;;
```

Comment:
double `;`

Reply:
Did you have a linter that told you that, or are you just a ninja?